policyengine_us/parameters/gov/irs/credits/lump_sum_distribution/expected_remaining_years_of_life.yaml

```diff
@@ -0,0 +1,488 @@
+description: The IRS provides the following multiples corresponding to the filer's expected remaining years of life. 
+metadata:
+  type: single_amount 
+  threshold_unit: year
+  period: year
+  amount_unit: /1
+  label: Expected remaining years of life
+  reference:
+      - title: Actuarial Values 3A Publication 1457 (Rev. 5-2009)
+        href: https://www.irs.gov/retirement-plans/actuarial-tables
+      # While the Ohio legal code reference the remaining years of life tables issues by the IRS  
```

Comment:
Are these values from 1457 or Ohio publications? Since it's in the IRS folder, it should only contain IRS parameters. If Ohio's parameters differ, let's add that as an Ohio policy parameter and file an issue to reconcile them.

Reply:
The legal code of Ohio points to the IRS parameters however the IRS values differ from the Ohio tax form values. Should we make this a Ohio parameter since the values used are derived from the Ohio tax form?

Reply:
Yes but before we do that could you help me confirm the values originate in the Ohio tax form? What are some examples of where they differ? Please include links.

Reply:
here is a link and picture for the acturail tables of the IRS
https://www.irs.gov/pub/irs-tege/table-s-2000cm.xlsx
<img width="777" alt="Screenshot 2024-01-28 at 9 31 13 PM" src="https://github.com/PolicyEngine/policyengine-us/assets/73863365/ed303f22-1745-42a0-a9ef-50f137d1cc57">

Here is the link and picture of the Ohio table values:
https://tax.ohio.gov/static/forms/ohio_individual/individual/2021/pit-it1040-booklet.pdf#page=29
<img width="1030" alt="Screenshot 2024-01-28 at 9 34 03 PM" src="https://github.com/PolicyEngine/policyengine-us/assets/73863365/29846caf-7765-487a-aa60-03e7a4112ca0">

The major confused is cased by this statemtn in this law:
https://codes.ohio.gov/ohio-revised-code/section-5747.055#:~:text=(G)%20A%20taxpayer%20sixty%2D,Code%2C%20as%20shown%20on%20an

Reply:
We have looked into the IRS publications but we cant seen to locate the adjustment factor specific for OH. 

We have created ohio specific vars in this PR and filed an issue consolidate